runtime/src/accounts_db.rs

```diff
@@ -2234,10 +2246,32 @@ impl AccountsDb {
             // `lock` released here
         };
 
+        let result = self.read_only_accounts_cache.load(pubkey, slot);
```

Comment:
Note that there's a known bug here:  that is currently reproducing very rarely, but may occur more often now that we've added extra logic between the `self.accounts_index.get()` and the load from cache/store: https://github.com/solana-labs/solana/pull/15454

Reply:
(nice way to ping me... lol, itoken and spl-token has been stealing my time... I try to fix #15454  tonight...)

Reply:
> (nice way to ping me... lol, itoken and spl-token has been stealing my time... I try to fix #15454 tonight...)

haha thanks! I think it's really close, you and Jeff might have to race for the conflicts 👊